kafka-ui-api/src/main/java/com/provectus/kafka/ui/config/auth/OAuthProperties.java

```diff
@@ -14,7 +15,16 @@ public class OAuthProperties {
   private Map<String, OAuth2Provider> client = new HashMap<>();
 
   @PostConstruct
-  public void validate() {
+  public void init() {
+    getClient().values().forEach((provider) -> {
+      if (provider.getCustomParams() == null) {
+        provider.setCustomParams(new HashMap<>());
```

Comment:
I've explicitly used this thing since `Map.of` returns an immutable collection and I'm not sure that something within spring won't try to modify these collections.
Fixed